src/style.css

```diff
@@ -0,0 +1,162 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;1,100;1,300;1,400;1,500;1,700&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.card-container {
+  display: flex;
+  position: relative;
+  width: 200px;
+  height: 408px;
```

Comment:
delete height from elements, height should be determined by the height of the content